sdk/resourcemanager/azure-resourcemanager-privatedns/CHANGELOG.md

```diff
@@ -8,6 +8,8 @@
 
 ### Bugs Fixed
 
+- Update CnameRecordSet no longer ignores changes to the cname property
```

Comment:
Update CnameRecordSet to make it no longer ignores changes to the cname property

(Just want to be more accurate)